Dockerfile

```diff
@@ -53,6 +53,8 @@ COPY . .
 # Required for Node < 15 (no native workspaces)
 RUN npx lerna link
 
+RUN npm run build
```

Comment:
We'll have to build outside of this Dockerfile, as you can only _build_ Directus using Node 15. This docker file copies the bundled code into the image, which is then being run

Reply:
Ahh ok now I understand why things were like they where. I will revert the related changes